src/app.js

```diff
@@ -1 +1,51 @@
-// write code here
+'use strict';
+
+/* eslint-disable no-console */
+
+const fs = require('node:fs');
+const path = require('node:path');
+
+const moveFile = (source, destination) => {
+  if (!fs.existsSync(source)) {
+    console.error(`Error: Source file does not exist: ${source}`);
+    return;
+  }
+
+  if (source === destination) {
+    return;
```

Comment:
It's good that you check if the source and destination are the same to avoid unnecessary operations. However, it might be helpful to log a message indicating that the operation was skipped because the source and destination are identical.